Jenkinsfiles/Jenkinsfile.build_ami

```diff
@@ -160,8 +141,15 @@ pipeline {
                 file(credentialsId: vault_pass, variable: 'vp')
               ]) {
                 sh """
+                  virtualenv -ppython2.7 venv
```

Comment:
Is there a reason not to use Python 3 to match our application servers?

Reply:
Hmm, I see now that this is copied from above. Still an interesting question though.

Reply:
I'll give this a shot, it shouldn't harm anything.

Reply:
Tested and it worked fine! 👍